### PR TITLE
Do not use OSP repo in image build

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: prepare-image.sh [hunk 1]
```diff
@@ -6,6 +6,10 @@ echo "install_weak_deps=False" >> /etc/dnf/dnf.conf
 # Tell RPM to skip installing documentation
 echo "tsflags=nodocs" >> /etc/dnf/dnf.conf
 
+if dnf repolist enabled | grep openstack-16-for-rhel-8-rpms &> /dev/null; then
```
Comment (dtantsur) [inline prepare-image.sh:9]: nit: could use `grep -q` instead of the redirection

File: packages-list.ocp [hunk 2]
```diff
@@ -12,6 +12,7 @@ openstack-ironic-python-agent >= 8.4.1-0.20220216141014.a83f384.el8
 parted
 psmisc
 python3-hardware-detect >= 0.29.0-0.20211122094056.7662a1d.el8
+python3-oslo-service >= 2.6.0-0.20220216002312.091fd65.el8
```
Comment (dtantsur) [inline packages-list.ocp:15]: Why is this needed? I'd prefer not to have package dependencies here.

File: prepare-image.sh (hunk 1 above)
Comment (elfosardo) [reply prepare-image.sh:9]: right

File: packages-list.ocp (hunk 2 above)
Comment (elfosardo) [reply packages-list.ocp:15]: this is just to trigger the image building, it matches the minimum version that we have in the repo
we can remove it once all the tests pass

**Comment by elfosardo** [comment]
/hold
we need the new ironic repo first

**Comment by elfosardo** [comment]
we'll change this in the config of the image itself
